src/style/list.css

```
@@ -1,41 +1,33 @@
-*{
-    margin:0;
-    padding: 0;
-}
-
-ul li{ margin: 4px 0; }
-
-li {
-    list-style:none; 
-    margin:1px;
-}
-
-.inner{
-    display: flex;
-    flex-direction: column;
-    flex-wrap: wrap;
-    height: 574px;
+.inner {
+    height: 560px;
     margin: 0 auto;
-    overflow: scroll;
+    overflow: hidden;
+    position: relative;
 }
-
-.inner .box_wrap li {
+.box_wrap{
+    display: flex;
+    flex-flow: row wrap;
     float: left;
-    padding: 0 1px;
-    width: calc(55% - 21px);
-  }
-
-.inner .box_wrap li .items {
-    border: 1px solid #e3e3e3;
-    padding: 9px 4px;
-    height: 140px;
+    justify-content: space-between;
+    margin: 15px
 }
-.items{
+.items { 
+    width: 155px;
+    height: 152px;    
+    border: 1px solid #e3e3e3;
     background: #FFFFFF;
     box-shadow: 0px 4px 24px rgba(0, 0, 0, 0.1);
-    border-radius: 15px;
+    border-radius: 15px;    
+    margin: 4px;
+}
+.items .list_form {
+    margin: 20px 15px;  
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 3;
+    overflow: hidden;
 }
```

```suggestion
   .items > div {
    margin: 20px 15px;  
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 3;
    overflow: hidden;
}
  ```
스타일를 부여한다 => 클래스를 추가한다는 무조건 지양하는게 좋지 않을까요..? 선택자를 최대한 많이 사용하는게 스타일을 유지보수할때 많이 편하긴합니다.